addons/crossword/src/presenter.js

```diff
@@ -590,9 +640,20 @@ function Addoncrossword_create(){
         return ModelValidationUtils.validateBoolean(model.blockWrongAnswers);
     };
 
+    presenter.upgradeModel = function (model) {
+        var upgradedModel = {};
```

Comment:
poprosiłbym, żebyś tą logikę przeniósł do oddzielnej metody, którą będziesz wywoływał w upgradeModel. Wiem, że wtedy masz metodę która jedyne co robi, to wywołuje inną metodę, ale byłoby to zrobione pod przyszłe propertiesy, oraz żeby być spójnym z rozwiązaniami z innych addonów.